packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -288,19 +289,13 @@ function idFromNumberOrSNode(node: SNode | number): number {
 function duplicateData(data: ISankeyChartData): ISankeyChartData {
   return {
     nodes: data.nodes.map(
-      // NOTE: We are not duplicating using `...node` because we want to omit the `sourceLinks` and `targetLinks`
       (node: SNode): SNode => ({
-        nodeId: node.nodeId,
-        name: node.name,
-        color: node.color,
-        borderColor: node.borderColor,
+        ...node,
```

Comment:
Have you looked at the comment on the left side of not using ...node.
Could you mention what has changed to allow using ...node for duplication.